tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java

```diff
@@ -543,10 +531,56 @@ public ClusterInfo examineBrokerClusterInfo() throws InterruptedException, MQBro
         return result;
     }
 
+    @Override
+    public AdminToolResult<ConsumerConnection> examineConsumerConnectionInfoConcurrent(String consumerGroup) throws InterruptedException, MQBrokerException, RemotingException, MQClientException {
+            return adminToolExecute(new AdminToolHandler() {
+            @Override
+            public AdminToolResult doExecute() throws Exception {
```

Comment:
You can replace with lambda

Reply:
This PR first ensures the correctness of the main process, and I will open a PR to deal with this matter later.